centos_7_py2py3/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM amd64/centos:7
+
+MAINTAINER Cosmo (hello@cloudify.co)
+
+RUN yum -y install python-devel gcc openssl git libxslt-devel libxml2-devel openldap-devel libffi-devel openssl-devel libvirt-devel python-netaddr
+
+RUN curl --silent --show-error --retry 5 https://bootstrap.pypa.io/get-pip.py | python
```

Comment:
This (with the following line) is a very odd way to spell yum install -y python-virtualenv

Reply:
Note: For this and the python3 one it might want a "pip install --upgrade pip setuptools" after the virtualenv has been created.

Reply:
didn't see any problem while testing

Reply:
The idea is to do that so that we reduce the chance of problems in future, by not needing to run a script interpreter on something that has been curled. yum has more built in checks to avoid problems.